src/tests/aes_tests.rs

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+use crate::{
+    aes::{
+        Aes128CbcPkcs7, Aes128Cfb, Aes128Cfb8, Aes128Ctr, Aes256CbcPkcs7, Aes256Cfb, Aes256Cfb8,
+        Aes256Ctr, Key,
+    },
+    traits::{Cipher, EncryptionKey},
+};
+use rand::{rngs::StdRng, SeedableRng};
+
+#[test]
```

Comment:
can we also add `wycheproof` tests for completeness?

Reply:
There doesn't seem to be wycheproof tests for any of the modes I've added so far. There's only tests for authenticated encryption, but I think it would be a good idea to add those along with some wycheproof tests.